pdf/utils.py

```diff
@@ -0,0 +1,23 @@
+"""
+Utility functions for XBlock
+"""
+
+from django.conf import settings
+
+
+def bool_from_str(str_value):
+    """
+    Converts string to boolean
+    """
+    return str_value.strip().lower() in [
+        'true'
+    ]
+
+
+def is_all_download_disabled():
+    """
+    Check if all download is disabled or not
+    """
+    return bool_from_str(
+        getattr(settings, 'PDFXBLOCK_DISABLE_ALL_DOWNLOAD', 'False')
```

Comment:
@nizarmah Let's not have the setting be a string when it can be a boolean. Maybe the original authors were constrained (or thought they were) and stored a string, but we don't have that problem through our settings variables, so we shouldn't mimic that behavior, especially when the variable is set in a place where every other variable is able to use booleans directly.

Reply:
I agree, I just was a bit confused about the yaml parser used with Ocim, but I just noticed that there are booleans there.

This should be fixed now :+1: 

btw I also applied the changes on the sandbox